sdk/lib/crt/except/arm/chkstk_asm.s

```diff
@@ -1,6 +1,6 @@
 /*
  * COPYRIGHT:         BSD - See COPYING.ARM in the top level directory
- * PROJECT:           ReactOS CRT librariy
+ * PROJECT:           ReactOS CRT library
  * PURPOSE:           Implementation of _chkstk and _alloca_probe
  * PROGRAMMER:        Timo Kreuzer (timo.kreuzer@reactos.org)
```

Comment:
Feel free to replace the programmers with your name, they are only there as a result of copy-pasting the file header and there was no actual code before.